.gitignore

```diff
@@ -3,3 +3,12 @@ client/js/*
 server/js/*
 .DS_Store
 *.out*
+
+# Build
+build/
+
+# editor-specific
+.vscode
```

Comment:
Just curious, what's in .vscode? Is it a project-specific config file for vscode? If so, might still be worth having it in source control so other people who use vscode can have the same project settings.